README.md

```diff
@@ -45,14 +56,14 @@ Alternatively, you can use build script dependencies and "apply plugin" syntax:
 buildscript {
     repositories {
         …
-        maven { url 'https://dl.bintray.com/kotlin/kotlinx' }
+        gradlePluginPortal()
     }
     dependencies {
-        classpath "org.jetbrains.kotlinx:kotlinx.benchmark.gradle:0.2.0-dev-20"
+        classpath "org.jetbrains.kotlinx:kotlinx-benchmark-plugin:0.3.0"
     }
 }
  
-apply plugin: 'kotlinx.benchmark'
+apply plugin: 'org.jetbrains.kotlinx.benchmark'
```

Comment:
We can replace this section about the legacy way to apply plugin with a link to https://plugins.gradle.org/plugin/org.jetbrains.kotlinx.benchmark

Reply:
Added plugin badge and removed this section